docs/tutorials/smart-contract/jsligo.mdx

```diff
@@ -45,7 +48,8 @@ It provides entrypoints that clients can call to change the value of that intege
 - The `decrement` entrypoint accepts an integer as a parameter and subtracts that integer from the value in storage
 - The `reset` entrypoint takes no parameters and resets the value in storage to 0
 
-After you deploy the contract, you or any other user can call it from a variety of Tezos clients and decentralized applications (dApps).
+After you deploy the contract, you or any other user can call it from various sources, including web applications, other contracts, and the Octez command-line client.
```

Comment:
depends on the permissions of course, not sure if that's worth mentioning there though, wdyt @timothymcmackin